py/selenium/webdriver/remote/remote_connection.py

```diff
@@ -375,7 +375,7 @@ def execute(self, command, params):
         LOGGER.debug("%s %s %s", command_info[0], url, str(trimmed))
         return self._request(command_info[0], url, body=data)
 
-    def _request(self, method, url, body=None, timeout=120):
```

Comment:
Is this internal or is it a breaking change?

Reply:
Internal change only since we reverted this from https://github.com/SeleniumHQ/selenium/pull/14578